hugo/content/contribute.md

```diff
@@ -3,83 +3,279 @@ title: Contribute
+Improve our Documentation and Website.
 
-## Ice authenticators
+## Documentation
 
-See {{< wiki Authenticators />}}. We’re happy for every authenticator and authenticator improvement you and the community can bring us!
+<!-- Todo: 
 
-## Positional Audio
+Add a concrete plan:
+- what documentation we want to have
+- what should be converted from the wiki
+- what is done & what is not
 
-Most of our positional audio plugins regularly break on game updates. To read on how you can help us update the plugins see the {{< wiki Pluginguide />}} page. You can even fix the update-issue forever for open source games, or if you want to add positional audio to your game, see {{< wiki Link />}}.
+- add issues and open pull requests
 
-## Mumble itself
+-->
+
+We plan to move all our documentation to the Website and Github Repo respectively.
+
+<!-- 
+
+**Todo:**
+
+- Convert parts of the Wiki into Markdown files
+- Rewrite parts of the documentation, so it is more recent, comprehensive and fits the style <!-- of the website, markdown files on github -->
+
+Right now you find most of Documentation in our
+[wiki](https://wiki.mumble.info).
+Any contributions are welcome, improving existing pages or creating new ones with useful content. Also see the {{< wiki "Category:Please_Improve" "Please Improve Category" />}}.
+
+The new places for documentation will be here:
+
+- [Mumble Github Repo - Docs](https://github.com/mumble-voip/mumble/tree/master/docs)
+
+- [Website's Github repo - Documentation](https://github.com/mumble-voip/mumble-www/tree/master/hugo/content/documentation)
+
+#### Documentation Issues & Pull requests
+
+- [Documentation Issues on Github](https://github.com/mumble-voip/mumble/labels/documentation)
+
+<!--
+Pull requests and issues:
+
+- [DOCS: Add Userguides to the documentation](https://github.com/mumble-voip/mumble/pull/4628) 
+
+- (Work-in-Progress) [C++ Guidelines & tools](https://github.com/mumble-voip/mumble/discussions/4261)
+
+-->
+
+
+---
+
+<!-- 
+**Ice methods**: 
+<!-- outdated? -->
+<!-- Is this here now: https://www.mumble.info/documentation/slice/1.3.0/html/Murmur.html  ? -->
+<!--
+We have a [generated documentation](http://mumble.sourceforge.net/slice/) for exposed Ice methods. Although probably only developers using the doc will be able to understand it, please give us a note on any inconsistencies, errors or needed clarifications if you find them.
+-->
+
+### External Documentation
+
+- **Wikipedia**: Someone that is not a development team member needs to maintain the [article(s) at Wikipedia](https://en.wikipedia.org/wiki/Mumble_%28software%29).
+
+
+## Website
+
+<!-- 
+Todo:
+Add more instructions?
+Here or at the Mumble-www repo.
+
+Such as:
+-Writing guidelines (markdown etc.)
+-Content (what is planned and needs improvement etc.?) 
```

Comment:
For this we should use issues in the www repo :point_up:

Reply:
I agree.